### DISCUSSION TITLE
simple gmsh test application

### DISCUSSION THREAD

**Discussion by fab6** [exadg/exadg#224] [Ideas]
Hi,
would it be possible to share a generic gmsh application case? Maybe some simple duct flow with an inlet and outlet. 
This would be great and make some initial settings a bit easier.
Fab

**Comment by necioglu** [comment]
Hi Fab,
I am currently working on reading an external mesh generated by cubit and exported in exodus file format. I am using this mesh for the Poisson/Sine application, but it would be generic code applicable to most applications. I will create a PR on this and can let you know. It will mostly rely on the [GridIn](https://www.dealii.org/current/doxygen/deal.II/classGridIn.html) class of dealii.
I hope this was helpful.
Fazil
